tests/system/datastore/test_v3io.py

```diff
@@ -51,42 +59,70 @@ def setup_class(cls):
         }
         with open(cls.test_file_path) as f:
             cls.test_string = f.read()
-        cls.test_dir_path = "/bigdata/v3io_tests"
-        cls.v3io_test_dir_url = "v3io://" + cls.test_dir_path
+        cls.test_dir = "/bigdata/v3io_tests"
+        cls.test_dir_url = "v3io://" + cls.test_dir
+        cls.run_dir = f"{cls.test_dir}/run_{uuid.uuid4()}"
```

Comment:
Here too, not very consistent with the f-strings. I see that this applies in several places so I won't comment on each and every one to avoid spamming this PR.